src/main/java/org/jboss/pnc/cleaner/orchApi/OrchClientProducer.java

```diff
@@ -18,6 +18,7 @@
 package org.jboss.pnc.cleaner.orchApi;
 
 import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.context.RequestScoped;
```

Comment:
Not used?

Reply:
Thanks! Good catch. Removed in the rebase.